src/components/biggive-page-section/biggive-page-section.tsx

```diff
@@ -1,4 +1,5 @@
 import { Component, Prop, h } from '@stencil/core';
+import { sectionStyle } from './biggive-page-section-prop-types';
```

Comment:
has to be imported not defined in this file because its also used in storybook, and there's an ESlint rule that says for a performance optimisation we're only supposed to have one export from each component file.